tor/core/user_interaction.py

```diff
@@ -328,3 +328,19 @@ def process_wrong_post_location(post):
             'Something went wrong with asking about a misplaced post; '
             'ignoring.'
         )
+
+def process_message(message, config):
+    if not isinstance(message, praw.models.Message):
```

Comment:
I'd just set `praw.models.Message` in line 132 as a statically typed param. `def process_message(message: praw.models.Message, config):`